core/src/components/reorder-group/test/basic/index.html

```diff
@@ -34,12 +34,22 @@
             <ion-reorder slot="end"></ion-reorder>
           </ion-item>
 
-          <ion-item button onclick="openAlert()">
-            <ion-label>
-              Item 2 (default ion-reorder)
-            </ion-label>
-            <ion-reorder slot="end"></ion-reorder>
-          </ion-item>
+          <ion-item-sliding>
+            <ion-item button onclick="openAlert()">
+              <ion-label>
+                Item 2 (default ion-reorder with ion-item-sliding)
+              </ion-label>
+              <ion-reorder slot="end"></ion-reorder>
+            </ion-item>
+            <ion-item-options side="start">
+              <ion-item-option onClick="favorite(item)">Favorite</ion-item-option>
```

Comment:
These functions don't exist (favorite, unread, etc). Can you remove them?

Reply:
Ah yeah good catch! Removed 😄